birdseye/server.py

```diff
@@ -80,12 +84,9 @@ def kill():
 
 
 def main():
-    try:
-        port = int(sys.argv[1])
-    except IndexError:
-        port = 7777
-
-    app.run(debug=True, port=port)
+    args = parser.parse_args()
+    host, port = args.host, args.port
```

Comment:
This unpacking seems unnecessary, why not just `app.run(..., port=args.port, host=args.host)`?

Reply:
Yeah, it's unnecessary, I'll fix it.